test/integration/dockerregistryclient_test.go

```diff
@@ -37,6 +79,8 @@ func TestRegistryClientConnect(t *testing.T) {
 }
 
 func TestRegistryClientConnectPulpRegistry(t *testing.T) {
+	requireReachableRegistry(t, "registry.access.redhat.com", "v2", false)
```

Comment:
Might as well make these strings `const` so there can't be drift between their uses.